packages/pebble-shared/svgs/lock.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1000pt" height="1600" viewBox="0 0 1000 1200"><path d="M837.5 440.625h-84.46V247.898C753.04 111.207 639.483 0 499.913 0 360.34 0 246.79 111.207 246.79 247.898v192.727H162.5c-77.54 0-140.625 63.086-140.625 140.625v478.129C21.875 1136.922 84.961 1200 162.5 1200h675c77.54 0 140.625-63.078 140.625-140.621V581.25c0-77.54-63.086-140.625-140.625-140.625zM340.54 247.898c0-85 71.495-154.148 159.374-154.148 87.879 0 159.375 69.148 159.375 154.148v192.727H340.54zm543.835 811.48c0 25.84-21.027 46.872-46.875 46.872h-675c-25.848 0-46.875-21.031-46.875-46.871V581.25c0-25.848 21.027-46.875 46.875-46.875h675c25.848 0 46.875 21.027 46.875 46.875zm0 0" fill="#000"/></svg>
```

Comment:
`lock` icon is missing keyhole that is present in `unlock` icon

Reply:
Added new icons for both.